geode-core/src/test/java/org/apache/geode/management/internal/configuration/mutators/ConfigurationMutatorFactoryTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.geode.management.internal.configuration.mutators;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Test;
+
+import org.apache.geode.cache.configuration.RegionConfig;
+
+public class ConfigurationMutatorFactoryTest {
+
+  @Test
+  public void generatesConfigurationMutatorFromConfigObjectType() {
+    ConfigurationMutatorFactory subject = new ConfigurationMutatorFactory();
+
+    RegionConfig config = new RegionConfig();
+    ConfigurationMutator configurationMutator = subject.generate(config);
```

Comment:
With the serviceInterface of `subject.generate(Class)` would be simpler than having to create `RegionConfig` instance

Reply:
While it would solve the problem of using the RegionConfig instance twice, I don't think it's "simpler", since you're not creating the RegionConfig instance, you're just using the one that's passed in.